src/ts/services/adapters/TyperighterAdapter.ts

```diff
@@ -69,7 +69,7 @@ class TyperighterAdapter implements IMatcherAdapter {
           return onRequestError({
             requestId,
             blockId: input.id,
-            message: `${response.status}: ${response.statusText}`,
+            message: `${response.url} responded with ${response.status} ${response.statusText}`,
             categoryIds,
             type: "AUTH_ERROR"
```

Comment:
Non-blocking, but it might be easier when graphing results to have this be the same type as the other errors? ie, `TYPERIGHTER_TELEMETRY_TYPE.TYPERIGHTER_ERROR`